datadog_checks_dev/datadog_checks/dev/tooling/cli.py

```diff
@@ -52,8 +49,8 @@ def ddev(ctx, core, extras, agent, here, color, quiet, debug):
     root = os.path.expanduser(config.get(repo_choice, ''))
     if here or not dir_exists(root):
         if not here and not quiet:
-            repo = 'datadog-agent' if repo_choice == 'agent' else 'integrations-{}'.format(repo_choice)
-            echo_warning('`{}` directory `{}` does not exist, defaulting to the current location.'.format(repo, root))
+            repo = 'datadog-agent' if repo_choice == 'agent' else f'integrations-{repo_choice}'
+            echo_warning(f'`{repo}` directory `{root}` does not exist, defaulting to the current location.')
```

Comment:
Note: another way to show "this is the output of a string" than using backticks is the `!r` specifier, e.g. `f"some {string!r}"` is formatted as `some 'string'`... We could *consider* migrating to that as a follow-up PR? 😄